code/__HELPERS/duplicating.dm

```diff
@@ -0,0 +1,96 @@
+///List of all vars that will not be copied over when using duplicate_object()
+GLOBAL_LIST_INIT(duplicate_forbidden_vars, list(
```

Comment:
Would be nice if we could use `NAMEOF()` here so that it compile errors when we remove a variable from the datum to keep this list current, but I can see how that might not be feasible if it's a global list?